src/sentry/lang/javascript/processor.py

```diff
@@ -900,8 +900,9 @@ def __init__(self, *args, **kwargs):
 
         # Component responsible for fetching the files.
         self.fetcher = Fetcher(
+            project=self.project,
             allow_scraping=organization.get_option("sentry:scrape_javascript", True) is not False
-            and self.project.get_option("sentry:scrape_javascript", True)
+            and self.project.get_option("sentry:scrape_javascript", True),
```

Comment:
I'd move it outside to `allow_scraping` var, as its hard to read now.

Reply:
I agree